src/lib/components/particles/svg-square/index.jsx

```diff
@@ -0,0 +1,19 @@
+//const fitBounds = (bounds) => {
+    
+export const SvgSquare = ({
```

Comment:
It looks like the `SvgSquare` component is an almost exact reimplementation of `<rect>`. I think we should remove it. (unless there is a good reason for it to exist?)